### DISCUSSION TITLE
How to add new third party provider for auth?

### DISCUSSION THREAD

**Discussion by AndersGerner** [supabase/supabase#1706] [Questions] [answered]
Bello Supabase people (In the best minion 🧑‍🦲 way of saying it :D )

ℹ️ **Background:** 
I am creating a project where I need to use Spotify as my third party auth provider.

❓ **Question:** 
Where can I see the current integrations for third party auth providers like Google or Bitbucket? 
I want to estimate the work to see if I could make a nice polished version and make it a PR to share it with everybody, rather just having it in my own project 😃 

Have a great day! 👋🏼 

Br,
Anders Gerner

**Comment by silentworks** [comment, marked as answer]
I think this is done via a third party open source library. You can probably open a PR on their repo and then make a request with the Supabase team to add it inside of their UI.

https://github.com/netlify/gotrue

**Reply by AndersGerner** [reply]
Oh! Go is not my strong suite - I had hoped for JS/TS 😸 

Thanks for your input @silentworks ! Much appreciated. 
I must hope for somebody else to expand the list of providers.

**Reply by soedirgo** [reply]
You can also just create an issue in https://github.com/supabase/gotrue @AndersGerner, then we or the community can add support for these over time :)

**Reply by AndersGerner** [reply]
Awesome! Thanks for the input @soedirgo ! I will create an issue in that repo. :) 

I will try and find another way to contribute. I'll try and see if there are any Danish translations missing besides the readme, which is already translated :)
